### PR TITLE
Print information about validators on startup

### PR DESCRIPTION
**Pull request by rolfyone**
- moved loggingConfigurator to update first during BeaconNode startup, so that log messages can be used in services

- centralised a function to shorten a BLSPublicKey

- made it so that the EncryptedKeystoreWriter and ValidatorLoader align on their shortened string of a public key to make it easier to find things.

Addresses #1762

Signed-off-by: Paul Harris <paul.harris@consensys.net>

### CODE REVIEW

File: util/src/test/java/tech/pegasys/teku/util/bytes/KeyFormatterTest.java [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+
+package tech.pegasys.teku.util.bytes;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+import org.apache.tuweni.bytes.Bytes;
+import org.junit.jupiter.api.Test;
+import tech.pegasys.teku.bls.BLSPublicKey;
+
+public class KeyFormatterTest {
```
Comment (macfarla) [inline util/src/test/java/tech/pegasys/teku/util/bytes/KeyFormatterTest.java:23]: would it be overkill to have a test for the null case?

File: validator/client/src/main/java/tech/pegasys/teku/validator/client/loader/ValidatorLoader.java [hunk 2]
```diff
@@ -46,13 +47,25 @@
...
     validators.putAll(createExternalSignerValidator(config));
 
-    LOG.debug(
-        "Loaded validators: {}",
-        () ->
-            validators.values().stream()
-                .map(Validator::getPublicKey)
-                .map(BLSPublicKey::toString)
-                .collect(joining(", ")));
+    if (validators.size() > 64) {
```
Comment (macfarla) [inline validator/client/src/main/java/tech/pegasys/teku/validator/client/loader/ValidatorLoader.java]: did you decide against the config flag on whether to print ?

**Review by macfarla — APPROVED** [review]

File: validator/client/src/main/java/tech/pegasys/teku/validator/client/loader/ValidatorLoader.java (hunk 2 above)
Comment (rolfyone) [reply validator/client/src/main/java/tech/pegasys/teku/validator/client/loader/ValidatorLoader.java]: i think it's probably overkill to start with . I haven't confirmed this with @ajsutton but i think hard coding a value will suffice for now.

File: util/src/test/java/tech/pegasys/teku/util/bytes/KeyFormatterTest.java (hunk 1 above)
Comment (rolfyone) [reply util/src/test/java/tech/pegasys/teku/util/bytes/KeyFormatterTest.java:23]: i can add one.

File: validator/client/src/main/java/tech/pegasys/teku/validator/client/loader/ValidatorLoader.java (hunk 2 above)
Comment (ajsutton) [reply validator/client/src/main/java/tech/pegasys/teku/validator/client/loader/ValidatorLoader.java]: Yeah I'd just hard code it.

Comment (ajsutton) [reply validator/client/src/main/java/tech/pegasys/teku/validator/client/loader/ValidatorLoader.java]: 64 is slightly weird number to pick mind you.  Maybe if it's 100 or above?

**Pull request merged**